src/app/home/pages/PlansPage/components/Wizard/NamespacesTable.tsx

```diff
@@ -97,9 +97,25 @@ const NamespacesTable: React.FunctionComponent = () => {
     const editedNamespace = values.editedNamespaces.find(
       (editedNS) => editedNS.id === namespace.id
     );
+    const { currentPlan } = useSelector((state: DefaultRootState) => state.plan);
+    let isTargetNamespaceNameSet = false
+    let setTargetNamespaceName = ""
+    if (currentPlan != null) {
+      for (var i of currentPlan.spec.namespaces) {
+        let ns = i.split(":");
+        if (ns.length > 1) {
+          if (ns[0] == namespace.name) {
+            isTargetNamespaceNameSet = true
```

Comment:
You probably don't need both of these variables, since `isTargetNamespaceNameSet` can be derived from whether or not `setTargetNamespaceName` is defined/truthy.